packages/react-scripts/config/tslint.json

```diff
@@ -0,0 +1,42 @@
+{
+  "defaultSeverity": "warning",
+  "rulesDirectory": "tslint-microsoft-contrib",
```

Comment:
[tslint-config-prettier](https://github.com/alexjoverm/tslint-config-prettier) and [tslint-react](https://github.com/palantir/tslint-react) would be nice defaults to have :+1:

Reply:
@andyrichardson which rules were you thinking from tslint-react? A lot of the rules seemed to be style prefer nces which we try to avoid.

Reply:
Fair point - `jsx-key` is already flagged by React so I guess the only other useful non-style option is `jsx-self-close`.

Reply:
This will expose tslint.json to the root, so technically we can add our own rules to the project, correct?

Reply:
@mohamedmansour just like ESLint, you will not be able to add your own rules. Added rules will strictly affect the editor output, not the build tools.